apps/crud/src/app/todoItem.store.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable, inject } from '@angular/core';
+import { ComponentStore, tapResponse } from '@ngrx/component-store';
+import { pipe, switchMap, tap } from 'rxjs';
+import { AppStore } from './app.store';
+import { TodoService } from './service/todo.service';
+import { Todo } from './interfaces/Todo';
+
+export interface TodoItemState {
+  todo?: Todo;
+  callState?: string;
+}
+
+@Injectable()
+export class TodoItemStore extends ComponentStore<TodoItemState> {
+  private todoService = inject(TodoService);
+  private appStore = inject(AppStore);
+
+  // replace with ngrxOnInit ? or constructor with super?
+  // probably should use ngrxOnInit but I'll leave this
```

Comment:
ngOnInit will never get call, you are inside an Injectable. Only ngOnDestroy is called. 

Use a contructor or ngrxOnInit. => that's why your patchState is not orking. Your state is never initialized